cypress/support/index.ts

```diff
@@ -20,3 +20,12 @@ Cypress.on('uncaught:exception', () => {
 });
 
 Cypress.Cookies.debug(true);
+
+// ignore fetch in command log
```

Comment:
Can you please explain to me why we need this?

Reply:
the fetch command is filling up the screen and make it harder to debug while checking errors on screenshots.